locales/es.json

```diff
@@ -1,4 +1,9 @@
 {
+  "alert": {
```

Comment:
Alert feels vague, especially in the locale file. Maybe `"weather_alert"` (or severity as I mentioned above)?